src/main/kotlin/wordle/domain/Game.kt

```diff
@@ -0,0 +1,49 @@
+package wordle.domain
+
+import java.time.LocalDate
+
+data class Game(val words: Words, val date: LocalDate) {
+
+    private var _count: Int = 0
+    private var _isOver: Boolean = false
+
+    val count: Int
+        get() = _count
+    val isOver: Boolean
+        get() = _isOver
+    val maxRound: Int
+        get() = MAX_ROUND
+
+    fun matchResult(playerWord: Word): Tiles {
+        require(words.contains(playerWord)) { "[ERROR] words.txt에 있는 단어를 입력해주세요." }
+        val tiles = mutableListOf<Tile>()
+        for ((index, spell) in playerWord.value.withIndex()) {
+            tiles.add(matchSpell(spell, index))
+        }
```

Comment:
`withIndex`라는게 있었군요. 꼭 파이썬의 `enumerate` 키워드를 보는 것 같네요.

```suggestion
        val tiles = playerWord.value.withIndex().mapIndexedNotNull { index, value -> matchSpell(value.value, index) }
```

오늘 제이슨이 이야기해주신 것 처럼, 이렇게 바꾸면 mutable list를 사용하지 않아도 될 것 같아요

Reply:
맞아요 이거 그때 못적었는데 언급해주셔서 감사해여!!